static/app/components/actions/resolve.tsx

```diff
@@ -134,61 +132,62 @@ class ResolveActions extends React.Component<Props> {
       ? t('Set up release tracking in order to use this feature.')
       : '';
 
-    const actionLinkProps = {
-      shouldConfirm,
-      message: confirmMessage,
-      confirmLabel,
-      disabled: disabled || !hasRelease,
+    const onActionOrConfirm = onAction => {
+      if (shouldConfirm) {
+        openConfirmModal({
+          onConfirm: onAction,
+          message: confirmMessage,
+          confirmText: confirmLabel,
+        });
+      } else {
+        onAction();
+      }
     };
 
+    const items = [
+      {
+        key: 'next-release',
+        label: t('The next release'),
+        details: actionTitle,
+        onAction: () => onActionOrConfirm(this.handleNextReleaseResolution),
+        showDividers: !hasRelease,
+      },
+      {
+        key: 'current-release',
+        label: latestRelease
+          ? t('The current release (%s)', formatVersion(latestRelease.version))
+          : t('The current release'),
+        details: actionTitle,
+        onAction: () => onActionOrConfirm(this.handleCurrentReleaseResolution),
+        showDividers: !hasRelease,
+      },
+      {
+        key: 'another-release',
+        label: t('Another existing release\u2026'),
+        onAction: () => this.openCustomReleaseModal(),
+      },
+    ];
+
     return (
-      <DropdownLink
-        customTitle={
-          <StyledActionButton
+      <DropdownMenuControlV2
+        items={items}
+        trigger={({props: triggerProps, ref: triggerRef}) => (
+          <DropdownTrigger
+            ref={triggerRef}
+            {...triggerProps}
+            size="xsmall"
             aria-label={t('More resolve options')}
-            disabled={!projectSlug ? disabled : disableDropdown}
             icon={<IconChevron direction="down" size="xs" />}
+            disabled={!projectSlug ? disabled : disableDropdown}
           />
+        )}
+        disabledKeys={
+          disabled || !hasRelease
+            ? ['next-release', 'current-release', 'another-release']
+            : []
         }
-        caret={false}
-        alwaysRenderMenu
-        disabled={!projectSlug ? disabled : disableDropdown}
-      >
-        <MenuHeader>{t('Resolved In')}</MenuHeader>
-
-        <MenuItemActionLink
-          {...actionLinkProps}
-          title={t('The next release')}
-          onAction={this.handleNextReleaseResolution}
-        >
-          <Tooltip disabled={hasRelease} title={actionTitle}>
```

Comment:
Removing tooltip here in favor of the `details` prop. This just makes the text more visible.

Before:
<img width="239" alt="Screen Shot 2022-01-28 at 12 46 58 PM" src="https://user-images.githubusercontent.com/44172267/151619184-f081e73e-2252-438e-a9c4-70cc2dfe37dd.png">

After:
<img width="334" alt="Screen Shot 2022-01-28 at 12 46 29 PM" src="https://user-images.githubusercontent.com/44172267/151619058-1745c171-c5a8-4608-b0a6-7ed01228622e.png">

Reply:
Ahh I LOVE this